pyBeam/include/geometry.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * pyBeam, a Beam Solver
+ *
+ * Copyright (C) 2018 Tim Albring, Ruben Sanchez, Rauno Cavallaro
+ *
+ * Developers: Tim Albring, Ruben Sanchez (SciComp, TU Kaiserslautern)
+ *             Rauno Cavallaro (Carlos III University Madrid)
+ *
+ * This file is part of pyBeam.
+ *
+ * pyBeam is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * pyBeam is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero
+ * General Public License along with pyBeam.
+ * If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#pragma once
+#include <math.h>
+
+#include "../include/types.h"
+
+class CNode {
+private:
+
+protected:
+
+  addouble *coord;
+
+public:
+
+  CNode(void);
```

Comment:
I added the CNode class so we can store the geometrical features of each node.

Reply:
Good, If the storing type is the SU2 one I will refer to that for my part, I'll give it a look soon